File: py_my_app/tests/tests_generate_events.py
from my_app.models.users import Users
from datashack.testing import datashack_test

@datashack_test
def test_users():
    test_data = {
        'id': 'test',
        'name': 'test', 
        'email': 'test@test.com',
    }

    # publish new event!
    Users.insert(test_data)

    # should be saved!
    assert Users.list()[0] == test_data

File: js_my_app/tests/tests_generate_events.js
const Users = require('./my_app/models/users');
const { datashack_test } = require('datashack/testing');

datashack_test(function test_users() {
  const test_data = {
    'id': 'test',
    'name': 'test', 
    'email': 'test@test.com',
  };

  // publish new event!
  Users.insert(test_data);

  // should be saved!
  assert.deepStrictEqual(Users.list()[0], test_data);
});
